config/config.go

```diff
@@ -105,6 +105,7 @@ func init() {
 	cfg.SetDefault("agent.flow.pcapsocket.min_port", 8100)
 	cfg.SetDefault("agent.flow.pcapsocket.max_port", 8132)
 	cfg.SetDefault("agent.flow.ebpf.kernel_scan_interval", 10)
+	cfg.SetDefault("agent.flow.ebpf.polling_rate", 1000000)
```

Comment:
maybe better `flow_per_minute`
You need to modify the default config file too, will be nice to add a comment to elaborate about the usage of this

Reply:
need to be adapted...